tests/qadence/test_matrices.py

```diff
@@ -76,99 +76,6 @@ def _calc_mat_vec_wavefunction(
 @pytest.mark.parametrize(
```

Comment:
Do you think we can also check the multiqubit gate matrices from the projector definition ? Ie. CNOT = Id x proj0 + X x proj1 ?